src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,21 @@
+package calculator
+
+enum class Operator(val symbol: String) {
+    PLUS("+"),
+    MINUS("-"),
+    MULTIPLY("*"),
+    DIVIDE("/");
+
+    fun apply(number1: Int, number2: Int) =
+        when (this) {
+            PLUS -> number1 + number2
+            MINUS -> number1 - number2
+            MULTIPLY -> number1 * number2
+            DIVIDE -> number1 / number2
+        }
+}
+
+fun String.findOperator(): Operator {
+    return Operator.values().firstOrNull() { it.symbol == this }
```

Comment:
`()`는 제거할 수 있을 것 같아요. 
```suggestion
    return Operator.values().firstOrNull { it.symbol == this }
```

확장함수 활용 👍💯
symbol에 따라서 Operator를 맵핑해주는 기능을 확장함수로 구현하신 이유가 있을까요? 🤔
개인적인 생각으로는 확장함수 보다는 companion object로 (java의 static 메서드)로 두는게 조금 더 보편적인 케이스라고 느껴져서요. 🤔

Reply:
확장함수라는 존재를 처음 익혔고, 이를 활용해보고 싶은 마음에 사용했습니다.
보통 확장함수는 언제 사용하나요?
+ 이 기회에 companion object에 대해 학습해보겠습니다. 
감사합니다!